plugins/apache_cassandra_logs.yaml

```diff
@@ -0,0 +1,110 @@
+version: 0.0.1
+title: Apache Cassandra
+description: Log parser for Apache Cassandra
+parameters:
+  - name: enable_system_logs
+    type: bool
+    default: true
+  - name: system_log_path
+    type: "[]string"
+    default: 
+    - "/var/log/cassandra/system.log"
+  - name: enable_debug_logs
+    type: bool
+    default: true
+  - name: debug_log_path
+    type: "[]string"
+    default:
+    - "/var/log/cassandra/debug.log"
+  - name: enable_gc_logs
+    type: bool
+    default: true
+  - name: gc_log_path
+    type: "[]string"
+    default:
+    - "/var/log/cassandra/gc.log"
+  - name: start_at
+    type: string
+    valid_values:
+     - beginning
+     - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_system_logs }}
+    filelog/system_logs:
+      include:
+        {{ range $fp := .system_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+        line_start_pattern: '[A-Z]+\s+\[[^\]]+\]'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'cassandra.system'
+      operators:
+        - type: regex_parser
+          regex: '(?P<severity>[A-Z]+)\s+\[(?P<type>[^\]]+)\]\s+(?P<timestamp>\d{4}-\d{2}-\d{2}\s+\d{2}:\d{2}:\d{2},\d{3})\s+(?P<message>.*\S)'
+          severity:
+            parse_from: attributes.severity
+        - type: time_parser
+          parse_from: attributes.timestamp
+          layout: '%F %T,%L'
+    {{ end }}
+
+    {{ if .enable_debug_logs }}
+    filelog/debug_logs:
+      include:
+        {{ range $fp := .debug_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+        line_start_pattern: '[A-Z]+\s+\[[^\]]+\]'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'cassandra.debug'
+      operators:
+        - type: regex_parser
+          regex: '(?P<severity>[A-Z]+)\s+\[(?P<type>[^\]]+)\]\s+(?P<timestamp>\d{4}-\d{2}-\d{2}\s+\d{2}:\d{2}:\d{2},\d{3})\s+(?P<message>.*\S)'
+          severity:
+            parse_from: attributes.severity
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%F %T,%L'
```

Comment:
It doesn't look like these logs and the system logs specify a time offset; We should probably add a timezone parameter for these timestamp parsers

Reply:
Done